failed in install btf2wit

**Describe the bug**

```bash
cargo install btf2wit
```

```   Compiling btfdump v0.0.1
error[E0308]: mismatched types
    --> /home/yao/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/btfdump-0.0.1/src/types.rs:1538:45
     |
1538 |         let c_str = unsafe { CStr::from_ptr(&strs[off as usize] as *const u8 as *const i8) };
     |                              -------------- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`
     |                              |
     |                              arguments to this function are incorrect
     |
     = note: expected raw pointer `*const u8`
                found raw pointer `*const i8`
note: associated function defined here
    --> /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/ffi/c_str.rs:256:25

For more information about this error, try `rustc --explain E0308`.
error: could not compile `btfdump` due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `btf2wit v0.1.1`, intermediate artifacts can be found at `/tmp/cargo-installpOisMF`
```

My Mac: `MacBook Pro (14-inch, 2021)`, `Version 13.2`, `M1 Max (ARM64,aarch64)`
My Develop: `Ubuntu 22.04.2 LTS (5.15.0-60-generic)`  in `Parallels Desktop 18 for Mac`

`cargo install btf2wit` in ubuntu.

rust verison:
```
rustc --version
rustc 1.67.1 (d5a82bbd2 2023-02-07)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -9,7 +9,7 @@ description = "Convert btf info to wit info"
 
 [dependencies]
 anyhow = "1.0.68"
-btfdump = "0.0.1"
+btfdump = { git = "https://github.com/anakryiko/btfdump", rev = "d9587c38" }
 clap = { version = "4.1.4", features = ["derive"] }
 memmap = "0.7.0"
 object = "^0.11.0"

